packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -538,6 +539,22 @@ const success = new stepfunctions.Fail(this, 'Fail', {
 });
 ```
 
+### Map
+
+A `Map` state can be used to run a step of steps for each element of an input array.
+A Map state will execute the same steps for multiple entries of an array in the state input.
+
+The result of a Map state is the transformed array after processing through iterator state
+machines.
```

Comment:
Not clear what you're saying here.

Reply:
I'd pulled it from our [reference documentation](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-stepfunctions.Map.html). Perhaps the initial sentence is also required for context.

Do we need to rewrite our definition of the Map State itself in that case? What do you think?

Reply:
Yeah, I don't think I understand what's said there either. Can we fix them both up?

Use https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-map-state.html

Reply:
sounds good